packages/tailwindcss/src/utilities.ts

```diff
@@ -73,35 +69,12 @@ export class Utilities {
     this.completions.set(name, groups)
   }
 
-  keys() {
-    return this.utilities.keys()
-  }
-
-  entries() {
-    return this.utilities.entries()
+  keys(kind: 'static' | 'functional') {
+    return Array.from(this.utilities.keys()).filter((key) => this.has(key, kind))
```

Comment:
I’m not sure how often this is called (if at all?) but we could probably avoid an array allocation here if we want

Reply:
yeah we can optimize this — was going to do that in a followup PR.  This is only used in intellisense so it matters less.

Reply:
Updated